README.md

```diff
@@ -110,6 +113,46 @@ Per-VDOM:
  * `fortigate_certificate_valid_to_seconds`
  * `fortigate_certificate_cmdb_references`
 
+Per-VDOM and Wifi-Client
+ * `fortigate_wifi_client_info`
+ * `fortigate_wifi_client_data_rate_bit_per_second`
+ * `fortigate_wifi_client_bandwidth_rx_bit_per_second`
+ * `fortigate_wifi_client_bandwidth_tx_bit_per_second`
+ * `fortigate_wifi_client_signal_strength_decibel`
+ * `fortigate_wifi_client_signal_noise_decibel`
+ * `fortigate_wifi_client_tx_discard_percentage`
```

Comment:
```suggestion
 * `fortigate_wifi_client_tx_discard_ratio`
```
And on all the other percentage metrics. It seems you are already converting them to a ratio (0.0-1.0) so it should only be a naming issue (and the descriptions should say Ratio instead of Percentage).

Reply:
Still need doc update it seems.